.gitignore

```diff
@@ -4,6 +4,7 @@ yarn-error.log
 .env
```

Comment:
Please don't need I have some concern and would like to understand what we are trying to achieve 🥺